### PR TITLE
change the version python and parameter reordering

### PR DESCRIPTION
**Pull request by shazaum**
two small fixes.
I used openbsd 7.2 for these fixes

### CODE REVIEW

File: create-ami.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/ksh
+#git@github.com:shazaum/aws-openbsd.git!/bin/ksh
```
Comment (ajacoutot) [inline create-ami.sh]: that shouldn't be part of the changes ;-)

Comment (shazaum) [reply create-ami.sh]: lol... that doesn't make sense...
I don't know how it got there.
I'll fix

Comment (shazaum) [reply create-ami.sh]: fixed

**Comment by ajacoutot** [comment]
Thanks, this looks good.
Could you resolve the conflicts?

**Comment by shazaum** [comment]
I realized that my fork was out of date

**Pull request closed without merging**